src/NSubstitute/Extensions/ExceptionExtensions.cs

```diff
@@ -247,21 +247,12 @@ private static object FromException(object value, Exception exception)
 
         private static Task TaskFromException(Exception ex)
         {
-#if NET45
-            return new Task(() => throw ex);
-#else
             return Task.FromException(ex);
```

Comment:
nit: we don't need a wrapper method after the change and this can be inlined

Reply:
you are right, done